mne/minimum_norm/tests/test_time_frequency.py

```diff
@@ -129,9 +130,204 @@ def test_tfr_with_inverse_operator(method):
+            n_cycles=2,
+            n_jobs=None,
+            return_plv=False,
+            method="dSPM",
+            prepared=True,
+        )
+
+    # error handling for multi-label and plv
+    with pytest.raises(RuntimeError, match="value cannot be calculated"):
+        source_induced_power(
+            epochs,
+            inv,
+            freqs,
+            labels,
+            baseline=(-0.1, 0),
+            baseline_mode="percent",
+            n_cycles=2,
+            n_jobs=None,
+            return_plv=True,
+            method="dSPM",
+            prepared=True,
+        )
+
+    # check multi-label handling
+    for ltype, lab_set in zip(("Label", "BiHemi"), label_sets):
+        vlen = vlen_lh if ltype == "Label" else vlen_lh + vlen_rh
+        # check overlapping verts error handling
+        with pytest.raises(RuntimeError, match="overlapping vertices"):
+            source_induced_power(
+                epochs,
+                inv,
+                freqs,
+                lab_set[1],
+                baseline=(-0.1, 0),
+                baseline_mode="percent",
+                n_cycles=2,
+                n_jobs=None,
+                return_plv=False,
+                method="dSPM",
+                prepared=True,
+            )
+
+        for ori in (None, "normal"):  # check loose and normal orientations
+            lbl = lab_set[0][0]
+
+            # check label=Label vs label=[Label]
+            no_list_pow = source_induced_power(
+                epochs,
+                inv,
+                freqs,
+                label=lbl,
+                baseline=(-0.1, 0),
+                baseline_mode="percent",
+                n_cycles=2,
+                pick_ori=ori,
+                n_jobs=None,
+                return_plv=False,
+                method="dSPM",
+                prepared=True,
+            )
+            assert no_list_pow.shape == (vlen, flen, tlen)
+
+            list_pow = source_induced_power(
+                epochs,
+                inv,
+                freqs,
+                label=[lbl],
+                baseline=(-0.1, 0),
+                baseline_mode="percent",
+                n_cycles=2,
+                pick_ori=ori,
+                n_jobs=None,
+                return_plv=False,
+                method="dSPM",
+                prepared=True,
+            )
+            assert list_pow.shape == (1, flen, tlen)
+
+            # check label=[Label, Label]
+            multi_lab_pow = source_induced_power(
+                epochs,
+                inv,
+                freqs,
+                label=lab_set[0],
+                baseline=(-0.1, 0),
+                baseline_mode="percent",
+                n_cycles=2,
+                pick_ori=ori,
+                n_jobs=None,
+                return_plv=False,
+                method="dSPM",
+                prepared=True,
+            )
+            assert multi_lab_pow.shape == (2, flen, tlen)
```

Comment:
You should also `assert_allclose` that the output for `[label]` is numerically the same as the output for `label` when you take the mean across vertices.